components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineView.kt

```diff
@@ -67,6 +68,10 @@ class GeckoEngineView @JvmOverloads constructor(
             rebind()
         }
 
+        override fun onFirstContentfulPaint() {
+            visibility = View.VISIBLE
+        }
```

Comment:
When I saw that I was wondering whether we could also set it to `VISIBLE` or `GONE` in `render()` based on whether the `Session` has already had its first contentful paint or not. But then I realized that we are passing `EngineSession` and not `Session`. 🤔